tools/cldr-unittest/src/org/unicode/cldr/unittest/TestCoverageLevel.java

```diff
@@ -664,4 +669,55 @@ private void checkBreakingLogicalGrouping(String localeId) {
             ++count;
         }
     }
+
+    public void testLogicalGroupingSamples() {
```

Comment:
Add some explicit tests for logical groups, to make sure we have all and only the right paths.